build.cake

```diff
@@ -123,6 +123,11 @@ Task("Default")
 
 void ExecuteProcess(string exePath, string arguments)
 {
+	if (IsRunningOnUnix())
```

Comment:
@ddunkin, `XmlDocMarkdown` uses `Context.Environment.Platform.IsUnix()`; maybe that's better?

Reply:
They're the same, `IsRunningOnUnix` is just shorthand.
https://github.com/cake-build/cake/blob/05b4d3f596defbdf5baecdb3712c9bc17f849b55/src/Cake.Common/EnviromentAliases.cs#L150-L157